src/Controller/Admin/MembershipApplicationCrud.php

```diff
@@ -101,6 +102,32 @@ public function acceptApplication(AdminContext $context)
         $member = $application->createMember(null);
         $member->generateNewPasswordToken();
         $em = $this->getDoctrine()->getManager();
+
+        if ($application->getPaid()) {
+            $init_payment = new ContributionPayment();
+            $init_payment->setMember($member);
+            $init_payment->setStatus(1);
+            $init_payment->setPaymentTime($member->getRegistrationTime());
+            $init_payment->setPeriodYear($member->getRegistrationTime()->format('Y'));
+            $init_payment->setPeriodMonthStart($member->getRegistrationTime()->format('n'));
+            switch ($member->getContributionPeriod()) {
+                case $member::PERIOD_MONTHLY:
+                    $init_payment->setPeriodMonthEnd($member->getRegistrationTime()->modify('+1 month')->format('n'));
+                    break;
+                case $member::PERIOD_QUARTERLY:
+                    $init_payment->setPeriodMonthEnd($member->getRegistrationTime()->modify('+3 month')->format('n'));
+                    break;
+                case $member::PERIOD_ANNUALLY:
+                    $init_payment->setPeriodMonthEnd($member->getRegistrationTime()->modify('+12 month')->format('n'));
```

Comment:
Ik denk dat `$member->getRegistrationTime()->modify('+12 month')` in place muteert, waardoor dit #217 veroorzaakt.